competencies/roles/_TechWideGeneral.md

```diff
@@ -122,3 +122,76 @@
         </ul></td>
     </tr>
 </table>
+
+### Interpreting Business Acumen
+
+Business acumen in a technology team context has two main dimension, one is core subject of
+our business (in the case of Properly today: customers buying and selling, the real estate industry)
+and the other is the technology business (role of cloud computing trends, major technology company business structure options)
+
+#### Examples of Emerging
+##### Real Estate
+Aware of the most relevant aspects of Canadian real estate in order to contextualize the 
+broader work that we are doing. Examples of the type of information: 
+- the agent system and fee structure (percent of cost, billed on sell side, billed from buyer agent to seller agent)
+- the reasons why buying first can be hard 
+  - risk of 2 homes and qualifying for 2 mortgages, unknown value of home you 
+  are selling 
+  - risk of 0 homes and not getting a suitable new home, managing the gap time
+- Understanding our system (estimating the sale price, discounting to unlock equity)
+
```

Comment:
I know this list isn't meant to be exhaustive, but I would suggest that an understanding of the fact that we source our listing data from third parties (implying need for interpretation, shaping, cleaning, etc) would be a valuable example of emerging competence here.